sass/main.scss

```diff
@@ -423,11 +423,25 @@ footer {
     }
 }
 
+@media screen and (max-width: 1440px) {
+    #hero {    
+        #bh7 {
+            margin-top: -217px;
+        }
+        #hero-text {
+            width: 57%;
```

Comment:
Width of the other div in the flex container wasn't adjusted -- also `57%` seems arbitrary